bigbluebutton-html5/imports/ui/components/user-list/user-list-content/generic-sidekick-content-button/component.tsx

```diff
@@ -42,14 +56,61 @@ const GenericSidekickContentNavButton = ({
     groupBySidekickMenuSection[section] = alreadySetArray;
   });
 
+  useEffect(() => {
+    if (pendingRenameCommand) {
+      setRenamedSection((prev) => {
+        const newObj = prev;
```

Comment:
Prevent changing the previous state directly, as react relies on shallow comparsion to determine whether to render the component:
```suggestion
        const newObj = { ...prev };
```